run_all.sh

```diff
@@ -8,7 +8,7 @@ echo "Making"
 eval "make" 
 
 
-for i in $(seq 1 14)
+for i in $(seq 1 15)
```

Comment:
Would it be possible to automatically compute this number from the tests themselves, so that you don't need to update it each time?

Or, we could use an alternative design: give each test a unique name (rather than a number), and then run every test in the directory.